File: PRG321/week02/p717N/SingleFamily.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace p717N10
{
    public class SingleFamily : Housing, IUnits
    {
        private double m_SqrFootage;
        private bool m_Garage;

        public double SquareFootage
        {
            set { m_SqrFootage = value; }
            get { return m_SqrFootage; }
        }

        public bool GarageAvailable
        {
            set { m_Garage = value; }
            get { return m_Garage; }
        }

        public override string ToString()
        {
            String old_Content = base.ToString();
            String new_Content = " - Square Footage = " + m_SqrFootage + " - Garage = ";
            if( m_Garage == true)
            {
                new_Content += " availabe";
            }
            else
            {
                new_Content += " Not Available";
            }
            return old_Content + new_Content;
        }
    }
}

File: PRG351_Java/Java_Mod_5/src/Java_Assignments/SingleFamily.java

package Java_Assignments;

/**
 *
 * @author CCSDuser
 */
public class SingleFamily extends Housing implements iUnits 
{
        private double m_SqrFootage;
        private boolean m_Garage;

        public void SetSquareFootage(double sqr)
        {
            m_SqrFootage = sqr;
        }
        public double GetSquareFootage()
        {
            return m_SqrFootage;
        }

        public void SetGarage(boolean value)
        {
            m_Garage = value;
        }
        
        public boolean GetGarage()
        {
            return m_Garage;
        }

        @Override
        public String ToString()
        {
            String new_Content = " - Square Footage = " + m_SqrFootage + " - Garage = ";
            if( m_Garage == true)
            {
                new_Content += " availabe";
            }
            else
            {
                new_Content += " Not Available";
            }
            return new_Content;
        }
}
